Packet++/src/DnsResourceData.cpp

```diff
@@ -164,7 +164,7 @@ GenericDnsResourceData::GenericDnsResourceData(const std::string& dataAsHexStrin
 	}
 }
 
-GenericDnsResourceData::GenericDnsResourceData(const GenericDnsResourceData& other)
+GenericDnsResourceData::GenericDnsResourceData(const GenericDnsResourceData& other) : IDnsResourceData()
```

Comment:
why is the compiler complaining here? 🤔

Reply:
This warning is a part of Wextra flag. `Base class ‘class pcpp::IDnsResourceData’ should be explicitly initialized in the copy constructor` I disabled extra flag after fix because test suite complains about lots of unused variable and prints lots of warnings.

Reply:
sounds good 👍